test/integration/suites/nested-rotation/intermediateA/server/server.conf

```diff
@@ -0,0 +1,34 @@
+server {
+	bind_address = "0.0.0.0"
+	bind_port = "8081"
+	registration_uds_path = "/tmp/spire-registration.sock"
+	trust_domain = "domain.test"
+	data_dir = "/opt/spire/data/server"
+	log_level = "DEBUG"
+    ca_ttl = "1h"
+	default_svid_ttl = "15s"
+}
+
+plugins {
+	DataStore "sql" {
+		plugin_data {
+			database_type = "sqlite3"
+			connection_string = "/opt/spire/data/server/datastore.sqlite3"
+		}
+	}
+	NodeAttestor "x509pop" {
+		plugin_data {
+            ca_bundle_path = "/opt/spire/conf/server/agent-cacert.pem"
+        }
+	}
+	KeyManager "memory" {
+		plugin_data = {}
+	}
+ 	UpstreamAuthority "spire" {
+ 	    plugin_data = {
+ 	        server_address      = "root-server"
+ 	        server_port         = 8081
+            workload_api_socket = "/opt/spire/sockets/workload_api.sock"
```

Comment:
weird indentation

Reply:
there was tabs with spaces, it is solved now